sql/core/src/test/resources/sql-tests/inputs/udf/pgSQL/udf-select_implicit.sql

```diff
@@ -0,0 +1,164 @@
+-- This test file was converted from select_implicit.sql
+-- Portions Copyright (c) 1996-2019, PostgreSQL Global Development Group
+--
+--
+-- SELECT_IMPLICIT
+-- Test cases for queries with ordering terms missing from the target list.
+-- This used to be called "junkfilter.sql".
+-- The parser uses the term "resjunk" to handle these cases.
+-- - thomas 1998-07-09
+-- https://github.com/postgres/postgres/blob/REL_12_BETA2/src/test/regress/sql/select_implicit.sql
+--
+
```

Comment:
Can you add the comment below here?

```
-- This test file was converted from pgSQL/select_implicit.sql 
-- [SPARK-28445] Inconsistency between Scala and Python/Panda udfs when groupby with udf() is used
-- TODO: We should add UDFs in GROUP BY clause when [SPARK-28445] is resolved.
```

Reply:
yes i will add these comments and update the PR.